dill/__init__.py

```diff
@@ -42,27 +42,28 @@
 
 ``dill`` is part of ``pathos``, a python framework for heterogeneous computing.
 ``dill`` is in active development, so any user feedback, bug reports, comments,
-or suggestions are highly appreciated.  A list of issues is located at https://github.com/uqfoundation/dill/issues, with a legacy list maintained at https://uqfoundation.github.io/project/pathos/query.
```

Comment:
Ok, this is mildly annoying, and hopefully temporary, but if you modify the module `__doc__` in this file, you also need to modify it on the README.md.  Hopefully, I'll reduce this to maintaining a single copy of the doc soon.  The reason for the two copies that I needed the doc in the different formats... and it wasn't as easy as it is now to convert between the formats.

Reply:
OK, applied the diff to it